<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RockPropertiesDialog</class>
 <widget class="QDialog" name="RockPropertiesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grid Math</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Shear Modulus (Mpsi):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Poisson Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>VP (ft/sec):</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leShearModulus"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="leYoungsModulus"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Youngs Modulus (Mpsi):</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Well</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>DEN (g/cm^3):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>VS (ft/sec):</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="lePoissonRatio"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbWell"/>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cbVP"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbVS"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="cbDEN"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Bulk Modulus (Mpsi):</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leBulkModulus"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbWell</tabstop>
  <tabstop>cbVP</tabstop>
  <tabstop>cbVS</tabstop>
  <tabstop>cbDEN</tabstop>
  <tabstop>leYoungsModulus</tabstop>
  <tabstop>leBulkModulus</tabstop>
  <tabstop>leShearModulus</tabstop>
  <tabstop>lePoissonRatio</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RockPropertiesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RockPropertiesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
